handsontable/src/core.js

```diff
@@ -4371,6 +4385,165 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     return instance.isLtr() ? 1 : -1;
   };
 
+  /**
+   * Returns the ShortcutManager's instance.
+   *
+   * @memberof Core#
+   * @since 12.0.0
+   * @function getShortcutManager
+   * @returns {object}
+   */
+  this.getShortcutManager = function() {
```

Comment:
In many places, I see the same chunks of code. What do you think about decreasing DRY by exposing ... hmm, let's say `getShortcutContext(contextName)`. The architecture of the shortcut module is designed in a way that most of the methods sit in the Context object, not in the Shortcut Manager itself.

So this:
```js
const shortcutManager = this.hot.getShortcutManager();
const gridContext = shortcutManager.getContext('grid');

// and then
gridContext.removeShortcutByGroup('my-group');
// or
gridContext.addShortcut(...);
```
to
```js
const gridContext = this.hot.getShortcutContext('grid')

gridContext.removeShortcutByGroup('my-group');
// or
gridContext.addShortcut(...);
```
or without assigning if there is a single method call
```js
this.hot.getShortcutContext('grid')
  .removeShortcutByGroup('my-group');
// or
this.hot.getShortcutContext('grid')
  .addShortcut(...);
```

Reply:
Same can be achieved by the current API `this.hot.getShortcutManager().getContext('grid')`

I find the current API clearer and easier to explain, albeit a bit verbose.

Reply:
I would add suggested method as it would be used quite often. @jansiegel WDYT?

Reply:
@wszymanski I don't think it's crucial, but it would be a nice addition.